src/main.d

```diff
@@ -93,6 +95,7 @@ int main(string[] args)
 			"display-sync-status", "Display the sync status of the client - no sync will be performed.", &displaySyncStatus,
 			"download-only|d", "Only download remote changes", &downloadOnly,
 			"disable-upload-validation", "Disable upload validation when uploading to OneDrive", &disableUploadValidation,
+			"dry-run", "Perform a trial sync with no changes made", &dryRun,
```

Comment:
In the man page there is an alternative `-d` (or `-n` as I suggested), but not supported here.